frontend/sozisel-app/src/components/utils/Button/Button.scss

```diff
@@ -0,0 +1,17 @@
+@use "../../../styles/color.scss";
+
+.button {
+  margin-top: 50px;
+  background-color: white;
+  border-radius: 10px;
```

Comment:
We should probably establish some kind of unified margins and paddings and put that in its own `scss` file so we can then reuse it and make everything look the same.